src/webview/leetCodeResultProvider.ts

```diff
@@ -34,19 +40,98 @@ class LeetCodeResultProvider implements Disposable {
         }
     }
 
-    private async provideHtmlContent(result: string): Promise<string> {
-        return `<!DOCTYPE html>
-            <html lang="en">
+    private parseResult(raw: string): Result {
+        try {
+            switch (raw[2]) {
+                case "√": {
```

Comment:
I'm not quite sure though.

Be careful about the icon we used here to check the result, because it may be different on different OS, see: https://github.com/skygragon/leetcode-cli/tree/master/icons.

On my Mac, it's `✘` and `✔`

Maybe we need to extract a logic to handle the different symbols from: https://github.com/jdneo/vscode-leetcode/blob/master/src/commands/list.ts#L46-L61

BTW, you can see that this is another limitation by using the `cli` instead of our own `lib`. 😢

Reply:
I checked the icon in the site, and covered all the relative chars in the regexp in the new commit.